deepchecks/vision/suites/default_suites.py

```diff
@@ -25,9 +28,87 @@
+           :widths: 50 50
+           :header-rows: 1
+
+           * - Check Example
+             - API Reference
+           * - :ref:`plot_vision_new_labels`
+             - :class:`~deepchecks.vision.checks.train_test_validation.NewLabels`
+           * - :ref:`plot_vision_similar_image_leakage`
+             - :class:`~deepchecks.vision.checks.train_test_validation.SimilarImageLeakage`
+           * - :ref:`plot_vision_heatmap_comparison`
+             - :class:`~deepchecks.vision.checks.train_test_validation.HeatmapComparison`
+           * - :ref:`plot_vision_train_test_label_drift`
+             - :class:`~deepchecks.vision.checks.train_test_validation.TrainTestLabelDrift`
+           * - :ref:`plot_vision_image_property_drift`
+             - :class:`~deepchecks.vision.checks.train_test_validation.ImagePropertyDrift`
+           * - :ref:`plot_vision_image_dataset_drift`
+             - :class:`~deepchecks.vision.checks.train_test_validation.ImageDatasetDrift`
+           * - :ref:`plot_vision_feature_label_correlation_change`
+             - :class:`~deepchecks.vision.checks.train_test_validation.FeatureLabelCorrelationChange`
+
+    Parameters
+    ----------
+    n_top_show: int, default: 5
+        Number of images to show, sorted by the similarity score between them.
+    label_properties : List[Dict[str, Any]], default: None
+        List of properties. Replaces the default deepchecks properties.
+        Each property is dictionary with keys 'name' (str), 'method' (Callable) and 'output_type' (str),
+        representing attributes of said method. 'output_type' must be one of:
+        - 'numeric' - for continuous ordinal outputs.
+        - 'categorical' - for discrete, non-ordinal outputs. These can still be numbers,
+          but these numbers do not have inherent value.
+        For more on image / label properties, see the :ref:`property guide </user-guide/vision/vision_properties.rst>`
+        - 'class_id' - for properties that return the class_id. This is used because these
+          properties are later matched with the VisionData.label_map, if one was given.
+    image_properties : List[Dict[str, Any]], default: None
+        List of properties. Replaces the default deepchecks properties.
+        Each property is dictionary with keys 'name' (str), 'method' (Callable) and 'output_type' (str),
+        representing attributes of said method. 'output_type' must be one of:
+        - 'numeric' - for continuous ordinal outputs.
+        - 'categorical' - for discrete, non-ordinal outputs. These can still be numbers,
+          but these numbers do not have inherent value.
+        For more on image / label properties, see the :ref:`property guide </user-guide/vision/vision_properties.rst>`
+    sample_size : int , default: 10_000
+        Max number of rows to use from each dataset for the training and evaluation of the domain classifier.
+    random_state: int, default: None
+        Random seed for all check internals.
```

Comment:
also here and in other suites- updates accordingly? 
(btw maybe "all relevant checks" / "all checks that use internally a random seed" / ...)